src/components/restaurant/restaurant-search/index.css

```diff
@@ -2,9 +2,22 @@
     bottom: 10px;
     left: 15px;
     font-size: 18px;
-    font-weight: bold;
 }
 
-.ttr-width-90 {
```

Comment:
Just to confirm if we are using ttr prefix or wd prefix for our self-defined stylesheet? I used .wd and I think the source code used .ttr. Which way do you think is better? We can then follow the naming convention.